### ISSUE TITLE
Terminology

### ISSUE THREAD

**Issue by thomasslee97** [NaSTAuk/graphics#7]
This probably won't be solvable until #4 and others are dealt with, but we definitely need to agree on what to call things.
- Clients?
- Controllers?
- Admin Panel?
- CG Panel?
- Data store?
Something to come back to later.

**Comment by benjaminkay93** [comment]
yeah i was getting confussed replying yesterday in my words, a glossary might help, good shout 🚀

**Comment by moschopsuk** [comment]
What is each graphical layer called? I just keep on referring to them as components but open for a suggestion. That helps better differentiate them from web components.

cgomponents 🤔

**Comment by thomasslee97** [comment]
I like the idea of calling each graphic a component. I think it would help the idea that this is a framework **and** library of components that can be used to build a graphics package for a show.

Works in the context of instructions/guides too: "For our football scores, we'll add the ___ component"

**Comment by benjaminkay93** [comment]
Thats a really good, shout, would the component encapsulate the admin bit too? so a component is the whole package? the CG view element and the bit the operator interacts with on an admin page?

**Comment by thomasslee97** [comment]
That was my thinking, a "component" would be an umbrella term for everything needed in terms of display/control. So adding a component to your setup for a show would mean adding the admin panel, CG, etc

**Comment by benjaminkay93** [comment]
yeah, I was thinking we could maybe have something that brings them in as pairs, abit of nice automagic for the users aha

**Comment by moschopsuk** [comment]
@thomasslee97 I agree with the term component, that will contain two child parts. One part being on the side to render for graphics. The other side being the controller.

**Comment by benjaminkay93** [comment]
Does #15 help?
